patient-view-be/src/main/java/spvproject/patientviewbe/service/ChiefComplaintService.java

```diff
@@ -0,0 +1,38 @@
+package spvproject.patientviewbe.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import spvproject.patientviewbe.dto.ChiefComplaintDTO;
+import spvproject.patientviewbe.model.ChiefComplaintModel;
+import spvproject.patientviewbe.model.CodeModel;
+import spvproject.patientviewbe.repository.ChiefComplaintRepository;
+import spvproject.patientviewbe.repository.CodeRepository;
+
+@Service
+public class ChiefComplaintService {
+
+	private final ChiefComplaintRepository cheifComplaintRepository;
```

Comment:
There's a small typo in the variable name, it should be called `chiefComplaintRepository`